spec/features/clearance/user_signs_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+require "support/features/clearance_helpers"
+
+feature "User signs in" do
+  scenario "with valid email and password" do
+    create_user "user@example.com", "user", "password"
+    sign_in_with "user@example.com", "password"
+
+    expect_user_to_be_signed_in
+  end
+end
+
+private
+
+def create_user(email, username, password)
```

Comment:
@teoljungberg @gabebw @JoelQ This helper is a copy of the one that is in clearance right now. Doesn't it feels a little weird to have these methods living in the user factory when the clearance gem is doing all the work?